setup/index.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
```

Comment:
Yes, I overlooked this as I was copying over and cutting a new branch. It's addressed [here](https://github.com/jsperf/jsperf.com/pull/58/files#diff-923ef69e5a84c767d6a31916439bc7acL112) in #58. Chose to ditch `let` since there was already a `var` immediately preceding and so we don't have to keep track of when and where to use `use strict`